src/constants.ts

```diff
@@ -1344,6 +1344,8 @@ export type TelemetryEvents = {
 	'launchpad/indicator/hidden': TelemetryEventData;
 	/** Sent when the launchpad indicator loads (with data) for the first time ever for this device */
 	'launchpad/indicator/firstLoad': TelemetryEventData;
+	/** Sent when a launchpad operation is taking longer than a set timeout to complete */
+	'launchpad/operation/slow': TelemetryEventData;
```

Comment:
Please fill in the specific typed payload here

Reply:
Done in commit: 665532eaf4fef604c72dc1cdeb4a9691fa93bda4